packages/menu/src/MenuItem/MenuItem.tsx

```diff
@@ -78,7 +78,7 @@ export const MenuItem = InferredPolymorphic<MenuItemProps, 'button'>(
     const baseProps = {
       ref,
       role: 'menuitem',
-      tabIndex: disabled ? -1 : undefined,
+      tabIndex: -1,
```

Comment:
why always -1?

Reply:
We only need the tab index for tabbing with the tab key, but since we got rid of it, we don't need to be able to tab organically since we're doing it programmatically on arrow click. This change isn't necessary, so I can revert if there are any issues with it.